gc3libs/compat/_functools.py

```diff
@@ -0,0 +1,78 @@
+__version__ = '0.1'
+
+try:
+    from functools import total_ordering
+except ImportError:
+
+    def total_ordering(cls):
+        """
+        Backport to work with Python 2.6
+        Class decorator that fills in missing ordering methods
+        Code from: http://code.activestate.com/recipes/576685/
+        """
+        convert = {
+            '__lt__': [
+                (
+                    '__gt__',
+                    lambda self, other: not (self < other or self == other)
```

Comment:
Why not use the [`operator` module](https://docs.python.org/2.6/library/operator.html) instead of lambda expressions?

Reply:
Copied from Python 2.7 source instead. It uses lambdas, but since it is actual python source then I think it's fine.